test/hotspot/jtreg/runtime/CommandLine/VMDeprecatedOptions.java

```diff
@@ -86,6 +91,16 @@ static void testDeprecated(String[][] optionInfo) throws Throwable {
             String match = getDeprecationString(deprecated[0]);
             output.shouldMatch(match);
         }
+
+        Object jfr = wb.getVMFlag("FlightRecorder");
+        if (jfr != null) {
+            String jfrOptionNames[] =  { "FlightRecorder" };
+            String jfrEexpectedValues[] = { "false" };
+            output = CommandLineOptionTest.startVMWithOptions(jfrOptionNames, jfrEexpectedValues);
+            output.shouldHaveExitValue(0);
+            String match = getDeprecationString(jfrOptionNames[0]);
+            output.shouldMatch(match);
+        }
```

Comment:
This is not the way to handle this. In the static block that initializes the set of flags to test you should do something like:

```
if (wb.isJFRIncluded()) {
  deprecated.add("FlightRecorder", false);
}
```
before line 70.

Reply:
Thank you for your comment. 
I fixed the test. Could you please review it again?